components/list/list.js

```diff
@@ -17,25 +19,49 @@ class List extends LitElement {
 			}
 		`;
 
-		const specialDividers = css`
+		const specialCases = css`
 			:host([divider-mode="none"]) {
 				--d2l-list-item-divider-bottom: none;
 				--d2l-list-item-divider-padding-bottom: 1px;
 				--d2l-list-item-divider-padding-top: 1px;
 				--d2l-list-item-divider-top: none;
 			}
-			:host([divider-mode="between"]) ::slotted(d2l-list-item:first-of-type),
 			:host([divider-mode="between"]) ::slotted(d2l-list-item:last-of-type) {
 				--d2l-list-item-divider-bottom: none;
 				--d2l-list-item-divider-padding-bottom: 1px;
+			}
+			:host([divider-mode="between"]) ::slotted(d2l-list-item:first-of-type) {
 				--d2l-list-item-divider-padding-top: 1px;
 				--d2l-list-item-divider-top: none;
 			}
+			:host([hover-effect]),
 			:host([divider-extend]) {
-				--d2l-list-item-content-padding: 0 18px;
+				--d2l-list-item-content-padding-side: 0.9rem;
+			}
+			:host([hover-effect]) ::slotted(d2l-list-item:hover) {
+				--d2l-list-item-divider-bottom: initial;
+				--d2l-list-item-divider-padding-bottom: initial;
+				--d2l-list-item-divider-padding-top: initial;
+				--d2l-list-item-divider-top: initial;
+			}
+			:host([selectable]) {
```

Comment:
I have mixed feelings/thoughts about this.  The list is not selectable, it is the list-items that are selectable.  By having this attribute/property misplaced (imo), I feel we are introducing some awkwardness where the list is trying to override/control rendering inside the items, etc.

Reply:
I agree, plus I really don't like always rendering the checkbox and just hiding it via CSS -- too many extra DOM nodes.

IMO I think we should go with the `selectable` attribute on the items. That way it actually opens the possibility of certain items not being selectable, which I know wasn't a requirement, but would be possible.

Reply:
And in reality, folks will almost always be rendering the items in a `forEach` look anyway, so moving the `selectable` attribute down to the item isn't any more code for them.

Reply:
So the main reason I put this on the list was to stop the ability have some items being selectable and others not. I haven't talked to Colin yet but having some selectable seems odd to me. I do need to add a disabled attribute if one cannot be selected.

I also, made it so `d2l-list` in PR 4 has the on change event so it is all tied into the parent.

I also, didn't like the extra DOM but when I thought about it it is only adding 2 the `label` and the `input` per item.

Reply:
So the main reason is to block the ability to make a weird looking list. So if you both think this doesn't matter as much I'll change it :)

The pros of less dom might be better. This change should be pretty easy to do.

Reply:
Yeah I totally get it. It's just sometimes preventing folks from doing weird things gets outweighed, and this might be one of those cases. Let's see how the conversation about single/multiple/disabled selection of items goes...

Reply:
- [ ] Add selectable to list-item

Reply:
I added the selectable to the list-item. Going to mark this as resolved as this seems like the best course of action so far.